lib/active_merchant/billing/gateways/ipg.rb

```diff
@@ -63,6 +63,7 @@ def store(credit_card, options = {})
       end
 
       def verify(credit_card, options = {})
+        options[:currency] = self.default_currency
```

Comment:
Hey @ajawadmirza -- does this value always have to be set to the default_currency, or could we allow users to pick their currency and only use the default if they don't specify another?